fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -425,6 +434,7 @@ aggDesc
     | BITMAP_UNION
     | PERCENTILE_UNION
     | REPLACE_IF_NOT_NULL
+    | AGG_STATE_UNION
```

Comment:
We can use an aggregation function here.

Reply:
Yes, `AGG_STATE_UNION` is a generic aggregate function which wraps all current aggregate functions which its input is an intermediate type.